src/main/java/baseball/service/BallService.java

```diff
@@ -0,0 +1,46 @@
+package baseball.service;
+
+import baseball.model.Ball;
+import baseball.model.BallNumbers;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BallService {
+
+    private static final String ILLEGAL_ARGUMENT = "입력값이 잘못되었습니다.";
+
+    public BallNumbers generateRandomBallNumbers() {
+        List<Ball> ballList = new ArrayList<>();
+        while (ballList.size() < 3) {
+            addBallToList(ballList, Randoms.pickNumberInRange(1, 9));
+        }
+
+        return new BallNumbers(ballList);
```

Comment:
```suggestion
        List<Ball> randomlyNumberedBalls = new ArrayList<>();
        while (ballList.size() < BaseballGame.MAX_BALL_COUNT) {
            randomlyNumberedBalls.add(createRandomlyNumberedBall()); 
            // createRandomlyNumberedBall 내부에 랜덤 숫자 선택 및 Ball 인스턴스 생성 로직 작성
        }

        return new BallNumbers(randomlyNumberedBalls);
```
메서드로 분리할 로직을 선택하는 건 각자의 취향이라서 정답은 없지만 제 방식으로 한 번 수정해봤으니까 참고만 해주세요!

https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming/
자바 변수명 네이밍 규칙 중 자료형을 변수 이름에 넣지 않는 걸 권고하는 내용이 있습니다! 위에 글 한 번 읽어보시면 좋을 것 같아요 ㅎㅎ

Reply:
글 첨부까지 해주시다니..!! 좋은 지적 정말 감사합니다!